app/models/dto.py

```
@@ -56,6 +136,22 @@ class AvailabilityRequest(BaseModel):
     neLat: float = Field(..., description="North-East Latitude")
     neLng: float = Field(..., description="North-East Longitude")
 
+    @field_validator("date")
+    @classmethod
+    def validate_date_regex(cls, value: str) -> str:
+        """YYYY-MM-DD 형식의 날짜 문자열만 허용"""
+        if not re.fullmatch(r"\d{4}-\d{2}-\d{2}", value):
+            raise ValueError("date must match YYYY-MM-DD")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)